app/build.gradle

```diff
@@ -68,6 +69,9 @@ dependencies {
     implementation rootProject.ext.AndroidXDependencies.constraintlayout
     implementation rootProject.ext.AndroidXDependencies.coreKtx
     implementation rootProject.ext.AndroidXDependencies.coreSplashscreen
+    implementation 'androidx.core:core-ktx:+'
+    implementation 'junit:junit:4.12'
+    implementation 'androidx.core:core-ktx:+'
```

Comment:
This is a duplicate, also we should move it to dependencies.gradle

Reply:
Thank you for reminding